### PR TITLE
Factions Update

### PR DESCRIPTION
**Pull request by MatrikSky**

### CODE REVIEW

File: Vindicta.Altis/Templates/Loadouts/Arma3_LDF/SF/A3_LDFs_TL.sqf [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+this addPrimaryWeaponItem "30Rnd_65x39_caseless_black_mag";
+this addWeapon "hgun_Pistol_heavy_01_green_F";
+this addHandgunItem "muzzle_snds_acp";
+this addHandgunItem "acc_flashlight_pistol";
+this addHandgunItem "optic_MRD_black";
+this addHandgunItem "11Rnd_45ACP_Mag";
+
+
+for "_i" from 1 to 2 do {this addItemToUniform "FirstAidKit";};
+for "_i" from 3 to 5 do {this addItemToVest "30Rnd_65x39_caseless_black_mag";};
```
Comment (billw2012) [inline Vindicta.Altis/Templates/Loadouts/Arma3_LDF/SF/A3_LDFs_TL.sqf]: This is weird, why from 3 to 5? that is the same as from 0 to 2? It doesn't randomly select 3 to 5 of them. IF you want random amount between 3 and 5 do this:
```suggestion
for "_i" from 0 to (3 + floor random 2) do {this addItemToVest "30Rnd_65x39_caseless_black_mag";};
```

Comment (MatrikSky) [reply Vindicta.Altis/Templates/Loadouts/Arma3_LDF/SF/A3_LDFs_TL.sqf]: Yep I noticed now thanks for letting me know, already fixed it.

**Comment by MatrikSky** [comment]
> Use of for loops looks suspicious, check the comment

Already fixed it thank for letting me know.

**Comment by MatrikSky** [comment]
SAPD Faction uses: 
- RHSUSAF https://steamcommunity.com/sharedfiles/filedetails/?id=843577117
- Ivory Car Pack https://steamcommunity.com/sharedfiles/filedetails/?id=1888644057
- San Andreas Police Uniform's https://steamcommunity.com/sharedfiles/filedetails/?id=1770527333

**Comment by billw2012** [comment]
Can you please update from our branch? The tests were not running correctly, I fixed it a few weeks ago but you need to update to get the fixes.

**Comment by MatrikSky** [comment]
I'll do it tomorrow morning right now I'm in bed with my phone. :P

**Comment by billw2012** [comment]
> I'll do it tomorrow morning right now I'm in bed with my phone. :P

Sorry ignore me, this problem is in latest as well!

**Comment by billw2012** [comment]
Just FYI we putting hold on accepting PRs until after next steam release (this week I hope). Soon as that is out we will start taking these.

**Comment by MatrikSky** [comment]
> YI

Yep I know already I've been following Dev-Chat. I'll just build up my changes in this PR for the future merge.

**Pull request closed without merging**